db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/FloatBufferedSumAggregator.java

```diff
@@ -35,55 +35,69 @@
 import com.splicemachine.db.iapi.sql.execute.ExecAggregator;
 import com.splicemachine.db.iapi.types.DataValueDescriptor;
 import com.splicemachine.db.iapi.types.NumberDataType;
+import com.splicemachine.db.iapi.types.SQLDouble;
 import com.splicemachine.db.iapi.types.SQLReal;
+import com.splicemachine.db.impl.sql.compile.Predicate;
+import org.apache.commons.lang3.mutable.MutableDouble;
 
 import java.io.IOException;
 import java.io.ObjectInput;
 import java.io.ObjectOutput;
 import java.util.Arrays;
+import java.util.Map;
+import java.util.TreeMap;
 
 /**
  * @author Scott Fines
  *         Date: 5/16/14
  */
 public class FloatBufferedSumAggregator extends SumAggregator {
-		private final float[] buffer;
```

Comment:
Isn't this a copy of DoubleBufferedSumAggregator now? What's the difference?

Reply:
@dgomezferro I didn't mention it in the JIRA, but I was considering removing FloatBufferedSumAggregator entirely from the source base, since it is no longer being used (it is never instantiated).  The only reason I left it in is in case we changed our minds a sprint or two from now and wanted to re-introduce it.  Do you think I should just go ahead and expunge it from the source entirely now?

Reply:
@msirek if it's not used then yes, definitely. Thanks!

Reply:
@dgomezferro I have gone ahead and removed all traces of FloatBufferedSumAggregator.